src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1 +1,36 @@
-export const TodoInfo = () => {};
+import React from 'react';
+import classname from 'classnames';
```

Comment:
Consider renaming the import from 'classname' to 'classNames' to follow the common naming convention for the 'classnames' library.

you can also use shorthand `cn`